templates/documentation/hydrology.html

```diff
@@ -0,0 +1,258 @@
+  were derived by applying the Variable Infiltration Capacity (VIC) model to
+  downscaled CMIP5 historical and projected climate data, using RCP 4.5 and RCP
+  8.5 emissions scenarios.
+</p>
+
+<p>
+  For more detailed model information, see the links to academic references and
+  source datasets included at the bottom of this page.
+</p>
+
+<h3>Service endpoints</h3>
+
+<h4>All hydrologic variables (point query)</h4>
+
+<p>
+  Query hydrologic variables for all models and scenarios. Monthly variable data
+  is averaged over each decade.
+</p>
+
+<table class="endpoints">
+  <thead>
+    <tr>
+      <th class="endpoint-label">Endpoint</th>
+      <th class="endpoint-url">Example URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Decadal means for all variables</td>
+      <td>
+        <a href="/hydrology/point/61.5/-147">/hydrology/point/61.5/-147</a>
+      </td>
+    </tr>
+  </tbody>
+  <tfoot>
+    <tr>
+      <td colspan="2">
+        Min/mean/max summaries of across all decades are available by appending
+        <code>?summarize=mmm</code> to the URL. <br />
+        CSV output is also available by appending <code>?format=csv</code> to
+        the URL. <br />
+        Chaining of multiple arguments is supported (e.g.,
+        <code>?summarize=mmm&format=csv</code>).
+      </td>
+    </tr>
+  </tfoot>
+</table>
+
+<h3>Output</h3>
+
+<h4>Decadal hydrology variables</h4>
+
+<p>Results from decadal queries will look like this:</p>
+
+<pre>
+  {
+    "ACCESS1-3": {
+      "rcp45": {
+        "apr": {
+          "1950-1959": {
+            "evap": "3",
+            "glacier_melt": "0",
+            "iwe": "131342",
+            "pcp": "120",
+            "runoff": "0",
+            "sm1": "10",
+            "sm2": "440",
+            "sm3": "0",
+            "snow_melt": "0",
+            "swe": "1348",
+            "tmax": "-6.5",
+            "tmin": "-20.7"
+          },
+          ...
+        },
+      ...
+    },
+    ...
+  },
+  ...
+},
+</pre>
+
+<p>The above output is structured like this:</p>
+
+<pre>
+{
+  &lt;model&gt;: {
+    &lt;scenario&gt;: {
+      &lt;month&gt;: {
+        &lt;decade&gt;: {
+          "evap": &lt;monthly sum of daily evapotranspiration, mm&gt;,
+          "glacier_melt": &lt;monthly sum of daily glacier ice melt, mm&gt;,
+          "iwe": &lt;monthly maximum of daily ice water equivalent, mm&gt;,
+          "pcp": &lt;monthly sum of daily precipitation, mm&gt;,
+          "runoff": &lt;monthly sum of daily surface runoff, mm&gt;,
+          "sm1": &lt;monthly mean of daily soil moisture in layer 1, mm&gt;,
+          "sm2": &lt;monthly mean of daily soil moisture in layer 2, mm&gt;,
+          "sm3": &lt;monthly mean of daily soil moisture in layer 3, mm&gt;,
+          "snow_melt": &lt;monthly sum of daily snowmelt, mm&gt;,
+          "swe": &lt;monthly maximum of daily snow water equivalent, mm&gt;,
+          "tmax": &lt;monthly mean of daily maximum air temperature at 2m, degrees C&gt;,
```

Comment:
```suggestion
          "tmax": &lt;monthly mean of daily maximum air temperature at 2m, &deg;C&gt;,
```

Reply:
I think this will yield a "degrees" symbol

Reply:
addressed with https://github.com/ua-snap/data-api/pull/358/commits/d22f5bf6bbd6cdd58d341a31c42b2f75f3ab724f